lang/java/reef-tests/src/test/java/org/apache/reef/tests/examples/TestHelloREEFMultiRuntime.java

```diff
@@ -53,11 +54,11 @@ public void testHelloREEFMultiRuntime() {
       // multi runtime can be tested on yarn only
       final Configuration driverConf = DriverConfiguration.CONF
               .set(DriverConfiguration.GLOBAL_LIBRARIES, EnvironmentUtils.getClassLocation(this.getClass()))
-              .set(DriverConfiguration.DRIVER_IDENTIFIER, "TEST_HelloREEF")
+              .set(DriverConfiguration.DRIVER_IDENTIFIER, "TEST_HelloREEFMultiRunitme")
               .set(DriverConfiguration.ON_DRIVER_STARTED, HelloMultiRuntimeDriver.StartHandler.class)
               .set(DriverConfiguration.ON_EVALUATOR_ALLOCATED, HelloMultiRuntimeDriver.EvaluatorAllocatedHandler.class)
               .build();
-      final LauncherStatus state = this.testEnvironment.run(driverConf);
+      final LauncherStatus state = new YarnMultiTestEnvironment().run(driverConf);
```

Comment:
I am not sure I actually need it. I can't use here the provided environment as I cannot detect multiruntime environment, so tried to make it same as other tests. I can remove it. WDYT?